Userland/Utilities/ps.cpp

```diff
@@ -101,6 +104,18 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
             warnln("Could not parse '{}' as a PID.", pid_string);
         return pid;
     }));
+    args_parser.add_option(make_list_option(uid_list, "Show processes with a matching user ID or login name. (Comma- or space-separated list.)", nullptr, 'u', "user-list", [&](StringView user_string) -> Optional<uid_t> {
+        if (auto uid = user_string.to_uint<uid_t>(); uid.has_value()) {
```

Comment:
This makes me ponder if it would be nicer to have `to_number<uid_t>` instead of forcing callers to decide if their type is "int" or "uint". :thonk:

Anyways, this PR looks great :)